docs/guide/personalization/recommendation_client.md

```diff
@@ -1,6 +1,6 @@
 # Recommendation client
 
-The [`ezrecommentation-client`](https://github.com/ezsystems/ezrecommendation-client) package 
+The [recommentation client package](https://github.com/ezsystems/ezrecommendation-client) package 
```

Comment:
Shouldn't be `personalization client package`?

Reply:
We haven't agreed on changing the name of the client. If @dkorostynski confirms I should rename Reco client to Perso client, I'll do it throughout